include/evm_runtime/evm_contract.hpp

```diff
@@ -85,11 +85,22 @@ class [[eosio::contract]] evm_contract : public contract
 
    [[eosio::action]] void setversion(uint64_t version);
 
+   [[eosio::action]] void updtgasparam(double kb_ram_price);
+
    // Events
    [[eosio::action]] void evmtx(eosio::ignore<evm_runtime::evmtx_type> event){
       eosio::check(get_sender() == get_self(), "forbidden to call");
    };
 
+   // Events
+   [[eosio::action]] void configchgdv1(eosio::ignore<uint64_t> gas_txnnewaccount, 
```

Comment:
The action should be called `configchange`. The versioning should be done through a variant similar to `evmtx`.

`config_change_v1` should have a `minimum_gas_price` and a struct containing the five gas parameters.